### PR TITLE
new pipeline for datahub helpcentre

### PR DESCRIPTION
**Pull request by markhigham**

### CODE REVIEW

File: datahub-helpcentre.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: datahub-helpcentre
+namespace: datahub
+scm: git@github.com:uktrade/data-hub-helpcentre.git
+environments:
+  - environment: dev
+    type: gds
+    region: eu-west-2
+    app: dit-staging/webops-dev/datahub-helpdesk-dev
```
Comment (lgarvey) [inline datahub-helpcentre.yaml]: Don't you want this in datahub-dev and not webops dev?  Could make things easier in future in terms of locating the app to be consistent.

File: datahub-helpcentre.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    vars: []
+    secrets: true
+    run: []
+  - environment: production
+    type: gds
+    region: eu-west-2
+    app: dit-services/datahub/datahub-helpdesk-production
```
Comment (lgarvey) [inline datahub-helpcentre.yaml]: really minor, but we typically drop the -production bit for the production app

Comment (markhigham) [reply datahub-helpcentre.yaml]: 👍

File: datahub-helpcentre.yaml (hunk 1 above)
Comment (markhigham) [reply datahub-helpcentre.yaml]: I don't have access to the datahub-dev space.

Comment (markhigham) [reply datahub-helpcentre.yaml]: Done.

**Pull request merged**